tests/unit/lms/authentication/_lti_policy_test.py

```diff
@@ -27,7 +27,7 @@ def test_it_returns_None_if_theres_no_lti_user(self, method_name, pyramid_reques
         policy = LTIAuthenticationPolicy()
         method = getattr(policy, method_name)
 
-        assert policy.authenticated_userid(pyramid_request) is None
+        assert method(pyramid_request) is None
```

Comment:
PyLint pointed out that the `method` fixture was unused. It turned out the test was wrong: it was meant to be calling `method` but wasn't